### PR TITLE
anchor link jump

### PR DESCRIPTION
**Pull request by Muzm**
rerereview

### CODE REVIEW

File: packages/helldoc-core/package.json [hunk 1]
```diff
@@ -23,6 +23,7 @@
     "@babel/plugin-proposal-class-properties": "^7.3.0",
     "@babel/preset-env": "^7.3.1",
     "@babel/preset-react": "^7.0.0",
+    "@types/react-router-hash-link": "^1.2.1",
```
Comment (fimars) [inline packages/helldoc-core/package.json]: move this `@type dependencies` to devDep and run the yarn again.

File: packages/helldoc-core/src/app/components/NavBar.tsx [hunk 2]
```diff
@@ -1,5 +1,5 @@
 import * as React from "react";
-import { Link } from "react-router-dom";
+import Link from "./Link";
```
Comment (fimars) [inline packages/helldoc-core/src/app/components/NavBar.tsx]: We don't need this Link Component, remove it.

File: packages/helldoc-core/src/app/components/Search.tsx [hunk 3]
```diff
@@ -1,6 +1,6 @@
 import * as React from "react";
 
-import { Link } from "react-router-dom";
+import Link from "./Link";
```
Comment (fimars) [inline packages/helldoc-core/src/app/components/Search.tsx]: same

File: packages/helldoc-core/src/app/components/Toc.tsx [hunk 4]
```diff
@@ -1,6 +1,6 @@
 import * as React from "react";
 import Heading from "../components/Heading";
-import { Link } from "react-router-dom";
+import Link from "./Link";
```
Comment (fimars) [inline packages/helldoc-core/src/app/components/Toc.tsx]: same

**Comment by fimars** [comment]
> Why this file (`_Navbar.scss`) name is upperCase ???

damn, I will fix it

File: packages/helldoc-core/src/app/components/Toc.tsx [hunk 5]
```diff
@@ -23,12 +23,12 @@ class HeadingLink extends React.PureComponent<HeadingLinkProps> {
     );
   }
 
-  private getLinkProps({ text }: HeadingData) {
+  private getLinkProps({ text, id }: HeadingData) {
     const clearText = text.replace(/\(.*\)/, "");
```
Comment (Nbsaw) [inline packages/helldoc-core/src/app/components/Toc.tsx]: What does `clearText` mean , can add some comment here ?  Look like want to remove `\` ?

**Pull request merged**